packages/notifi-wallet-provider/lib/context/NotifiWallets.tsx

```diff
@@ -1,6 +1,4 @@
 import { Abstraxion, useModal } from '@burnt-labs/abstraxion';
-import '@burnt-labs/abstraxion/dist/index.css';
-import '@burnt-labs/ui/dist/index.css';
```

Comment:
@ankush-notifi I removed the .css import statement  for now (I think it is part of MVP-5036).
Still need your help complete the part of README.md. (How to setup zion on dapp when being adopted)